test/mock_auth/server.py

```diff
@@ -0,0 +1,141 @@
+import os
+import sys
+import json
+import jsonschema
+import traceback
+import flask
+from jsonschema.exceptions import ValidationError
+
+# Load the endpoints data, the schema, and validate the structure
+
+# For validating every config file
+with open('endpoint_schema.json') as fd:
+    endpoint_schema = json.load(fd)
+
+if not os.path.exists('/config'):
+    sys.stderr.write('Path not found: /config\n')
+    sys.exit(1)
+
+endpoints = []
+catalog_mock_auth = ['auth_admin.json', 'auth_invalid.json', 'auth_missing.json', 'auth_non_admin.json']
```

Comment:
It would be good to put a comment here explaining that the order is being preserved here in order to ensure the routes capture the requests in the correct order

Reply:
👍

Reply:
I did add a comment. For some reason, it is not showing up here, but you can see it in the changed files.

Reply:
I see "# The order is preserved here to ensure routes capture requests correctly." this is not very descriptive and not really telling the story of why this needs to be here.

Reply:
Comment added. Please let me know if it makes sense now.

Reply:
LGTM!